selectBoard.php

```diff
@@ -218,6 +245,12 @@
   </div>
 </div>
 
+<script>
+function dynamicModal(str)
+{
+$("#shareBoardFrame").attr("src", "https://mansci-db.uwaterloo.ca/~wmmeyer/dev_gaurav/shareBoardCourse.php?id="+str); //this link will need to change once deployed
```

Comment:
Just a reminder on github (I see the comment in the code) that we need to point to the root.